armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -0,0 +1,492 @@
+# Copyright 2019 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""enable component-wise axial expansion for assemblies and/or a reactor"""
+
+from statistics import mean
```

Comment:
What's the difference because this package and numPy? Is statistics apart of the standard library?

Reply:
This is part of the Python standard library?  And it's been in there since Python v3.4? 

That's crazy, I haven't seen it.  Nice.

Reply:
Yeah really good to know! Just wanted to check but it's good fine to use the std lib for things. Just wanted to make sure a new dependency wasn't being added.